From cbb88a514ba537de59a13fa70ebc86a9ddefef53 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Wed, 1 May 2019 21:44:11 +0000
Subject: [PATCH] CMakeLists.txt: don't check for signals Boost library

deprecated signals library was dropped in 1.69.0 release:
https://www.boost.org/users/history/version_1_69_0.html

Signals (v1) is now removed. Its removal was announced in 1.68 and its deprecation was announced in 1.54. Boost 1.68 is the last release that provides this library. Users are encouraged to use Signals2 instead. The Boost community thanks Douglas Gregor for his work on Signals which served its users well and which also inspired Signals2.

This component is already using Signals2 so just drop the check for old
unused component.

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 CMakeLists.txt | 2 +-
 README.md      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 86e757b..8f3fc1f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,7 +46,7 @@ pkg_check_modules(PMLOG REQUIRED PmLogLib)
 include_directories(${PMLOG_INCLUDE_DIRS})
 webos_add_compiler_flags(ALL ${PMLOG_CFLAGS_OTHER})
 
-find_package(Boost REQUIRED COMPONENTS signals)
+find_package(Boost REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 webos_add_compiler_flags(ALL ${Boost_CFLAGS_OTHER})
 add_definitions(-DBOOST_BIND_NO_PLACEHOLDERS)
diff --git a/README.md b/README.md
index 73b5cef..a1d4d77 100644
--- a/README.md
+++ b/README.md
@@ -18,7 +18,7 @@ Below are the tools and libraries required to build appinstalld2:
 * webosose/luna-service2
 * webosose/libpbnjson
 * webosose/pmloglib
-* Boost-signals
+* Boost
 * webosose/pmtrace
 
 Copyright and License Information
